app/locales/nl/translations.js

```diff
@@ -174,6 +177,11 @@ export default {
         afke2: 'Wat mij enorm aanspreekt in de vereniging is dat iedereen welkom is. Er zijn leden uit allerlei verschillende kerken en ook niet-gelovige mensen, maar iedereen gaat met elkaar om en iedereen heeft het gezellig met iedereen. Zelf ben ik niet gelovig, maar ik voel me zeker helemaal op mijn plek binnen Alpha. De goede gesprekken, de vriendschappen en de epische borrels en feesten in onze sociëteit zorgen ervoor dat Alpha een superinvulling is van mijn studententijd in Enschede!'
       }
     },
+    lustrum: {
+      lustrumBar: {
+        signUp: 'Schrijf je in via de mail!'
```

Comment:
Gaan we niet gewoon een google form gebruiken? Want dan is het houden op "Schrijf je in" veel logischer.

Reply:
Lol, je bent veel te snel